components/Buttons/tests/unit/ButtonsTests.m

```diff
@@ -521,6 +527,35 @@ - (void)testTitleFontForStateFallbackBehavior {
   }
 }
 
+/** Test dynamic type responds to @c UIContentSizeCategoryDidChangeNotification. */
+- (void)testDynamicTypeBehavior {
```

Comment:
Nit/naming: Please be more specific in this test name. The comment above should probably just be the test name. Something like, `testContentSizeNotificationTriggersDynamicTypeFontChange`.

Reply:
This has been removed in favor of snapshot test.